### ISSUE TITLE
[BUG] Misplaced native terminal cursor in dialog boxes with tabs

### ISSUE THREAD

**Issue by shindere** [anthropics/claude-code#42684]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

When a dialog box with tabs opens, the cursor remains at the bottom of the screen. This is the continuation of #36765 and #39900.

### What Should Happen?

The native cursor of the terminal should be put on the title of the selected tab. For instance, in the help dialog box the cursor should be on the g of general.

### Error Messages/Logs

```shell

```

### Steps to Reproduce

Simply run the /help command.

### Claude Model

None

### Is this a regression?

Yes, this worked in a previous version

### Claude Code Version

2.1.90 (Claude Code)

### Platform

Other

### Operating System

Ubuntu/Debian Linux

### Terminal/Shell

Other

### Additional Information

Like in the two other issues, this bug report is platform-independant and affects all users of assistive-technologies, especially users of screen readers.
